src/oaklib/implementations/ontoportal/ontoportal_implementation_base.py

```diff
@@ -126,6 +127,17 @@ def labels(self, curies: Iterable[CURIE], allow_none=True) -> Iterable[Tuple[CUR
     def annotate_text(
         self, text: str, configuration: TextAnnotationConfiguration = None
     ) -> Iterator[TextAnnotation]:
+        """
```

Comment:
note this is repetitious with the interface. Big danger of things getting out of sync.

we need to decide on a general pattern for documenting implementations. Thus far I have omitted docstrings. But if we do include them they should add information and not repeat.

We could say something like "implements annotate_text from text_annotator by calling the /annotate/ endpoint using ontoportal client"